create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/test/utils/ScopusGenerator.java

```diff
@@ -165,9 +171,56 @@ private static HeadTp randomHeadTp() {
     private static CitationInfoTp randomCitationInfo() {
         var citationInfo = new CitationInfoTp();
         citationInfo.setAuthorKeywords(randomAuthorKeywordsTp());
+        citationInfo.getCitationType().add(randomCitationType());
         return citationInfo;
     }
 
+    private static CitationTypeTp randomCitationType() {
+        var citationType = new CitationTypeTp();
+        citationType.setCode(articleCitationTypeAtt());
+        return citationType;
+    }
+
+    private static CitationtypeAtt articleCitationTypeAtt() {
+        return CitationtypeAtt.AR;
+    }
+
+    private static CitationTypeTp randumUnsupportedCitationType() {
+        var citationType = new CitationTypeTp();
+        citationType.setCode(randomUnSupportedCitationTypeAtt());
+        return citationType;
+    }
+
+    private static CitationtypeAtt randomUnSupportedCitationTypeAtt() {
+        List<CitationtypeAtt> citationTypeAttList = new ArrayList<>(
```

Comment:
```
return randomElement(CitationtypeAtt.values());
```

Reply:
I am not sure this method is necessary if you use the parametrized test

Reply:
randomElement(CitationtypeAtt.values()) would also return the supported type CitationType.AR.

Reply:
Using parametrized test